docs/source/concept_models/vivarium_census_synthdata/concept_model.rst

```diff
@@ -1538,15 +1541,14 @@ survey iterations.
 
 
 **Data Errors/Noise** 
-In the future, we will add a noise function designed to replicate 
-missed or incorrect data. This includes incorrect name spelling, 
-addresses, age, and person swaps. 
 
-Some errors may be introduced within the simulation, such as 
-duplicate people or swapped people. Others may be introduced 
-at the time of sampling, such as name and address misspellings. 
+.. todo::
+    Engineers: these things are not yet defined and might be edited later - they should be parameterizable 
 
-These are not currently modeled. 
+    - Omission rates by survey, oversampling might be needed to avoid rerunning 
```

Comment:
Technically omission wouldn't require any oversampling, right?

That said, I think you should specify here a maximum number we might want to sample, so that we could change the sample rate up to that number without re-running.

Reply:
Sure, changed language and gave a number here